WordPress/Classes/ViewRelated/Blog/SharingDetailViewController.m

```diff
@@ -96,10 +107,19 @@ - (NSString *)tableView:(UITableView *)tableView titleForFooterInSection:(NSInte
     if (section == 0) {
         return NSLocalizedString(@"Allow this connection to be used by all admins and users of your site.", @"");
     }
-    if (section == 1 && [self.publicizeConnection isBroken]) {
-        NSString *title = NSLocalizedString(@"There is an issue connecting to %@. Reconnect to continue publicizing.", @"Informs the user about an issue connecting to the third-party sharing service. The `%@` is a placeholder for the service name.");
-        return [NSString stringWithFormat:title, self.publicizeConnection.label];
+
+    if (section == 1) {
```

Comment:
I would suggest extracting each logical branch to a separate method. That way `titleForFooterInSection` would be easier to read and parse, which might also make it. in the long term, more difficult to get wrong.